oauthenticator/generic.py

```diff
@@ -75,6 +76,8 @@ def authenticate(self, handler, data=None):
         code = handler.get_argument("code")
         # TODO: Configure the curl_httpclient for tornado
         http_client = AsyncHTTPClient()
+        
+        tls_verify = ast.literal_eval(os.environ.get('OAUTH2_TLS_VERIFY', 'True'))
```

Comment:
A few things:

- this should be a Bool(True, config=True) traitlet above, not a local variable
- instead of `literal_eval`, check for specific values in the env, e.g. (v.lower() in {'true', '1'})

Reply:
Thank you for the review. I am new to Python so I need to read things about traitlets first.

Reply:
Ok I implemented a new traitlet, but I somehow edited an older version of the file so the commits are a bit messed up.